examples/typescript-backbone/readme.md

```diff
@@ -26,7 +26,7 @@ Articles and guides from the community:
 Get help from other TypeScript users:
 
 * [TypeScript on StackOverflow](http://stackoverflow.com/questions/tagged/typescript)
-* [Forums](http://typescript.codeplex.com/discussions)
+* [Forums](https://github.com/Microsoft/TypeScript/issues)
```

Comment:
Other than the issues, all we have is a Gitter channel. Discussions do also occur over our issue tracker. Perhaps we can patch this up at a later change?